qtmodern/windows.py

```diff
@@ -169,6 +168,39 @@ def setWindowTitle(self, title):
         super(ModernWindow, self).setWindowTitle(title)
         self.lblTitle.setText(title)
 
+    def _setWindowButtonState(self, button: QToolButton, state: bool):
+        button.setVisible(state)
```

Comment:
Did you test if this interfers with how we toggle between `btnRestore` and `btnMaximize`?
I am thinking because we toggle them by the visibility we might need to add logic to check for the flags in `on_btnRestore_clicked` and `on_btnMaximize_clicked`.

Reply:
added